daemon/execdriver/native/tmpfs.go

```diff
@@ -0,0 +1,45 @@
+package native
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func genTmpfsPremountCmd(tmpDir string, fullDest string, dest string) []configs.Command {
+	var premount []configs.Command
+	if _, err := os.Stat("/bin/tar"); os.IsNotExist(err) {
+		logrus.Warn("/bin/tar is not available for tmpfs mount")
+		return premount
+	}
+	tarFile := fmt.Sprintf("%s/%s.tar", tmpDir, strings.Replace(dest, "/", "_", -1))
+	if _, err := os.Stat(fullDest); err == nil {
+		premount = append(premount, configs.Command{
+			Path: "/bin/tar",
```

Comment:
You're hardcoding pathes for both `tar` and `rm`. On my system `rm` is in `/bin`. You need to use `exec.LookPath`.